great_expectations/data_context/data_context/data_context.py

```diff
@@ -33,6 +33,13 @@
 from great_expectations.datasource.new_datasource import BaseDatasource
 from great_expectations.rule_based_profiler.rule_based_profiler import RuleBasedProfiler
 
+# BDIRKS: All ZEP logic should be moved to abstract_data_context.py. However, the class
+# currently BaseDataContext doesn't call super().__init__().
+from great_expectations.zep.metadatasource import _SourceFactories, Datasource as ZepDatasource
+import great_expectations.zep.postgres_datasource
```

Comment:
As per the slack thread.
Eventually each `DataContext` does call `super().__init__()`. 
`BaseDataContext` is somewhat of a wrapper around a real data context, and each of those instances does call `super().__init__()

https://github.com/great-expectations/great_expectations/blob/develop/great_expectations/data_context/data_context/base_data_context.py#L246-L271

Reply:
Sounds good. It looks like I will need to do a little plumbing in this file to plumb the variable on the abstract object to this one.